airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/report_streams.py

```diff
@@ -102,11 +102,18 @@ class ReportStream(BasicAmazonAdsStream, ABC):
     # Format used to specify metric generation date over Amazon Ads API.
     REPORT_DATE_FORMAT = "%Y%m%d"
     cursor_field = "reportDate"
+    #Maximum number of retries that Airbyte attempt to fetch data.
+    REPORT_GENERATION_MAX_RETRIES = 5
 
     def __init__(self, config: AmazonAdsConfig, profiles: List[Profile], authenticator: Oauth2Authenticator):
         self._authenticator = authenticator
         self._session = requests.Session()
         self._model = self._generate_model()
+        if(config and config.report_wait_timeout and config.report_wait_timeout > 30):
+            self.REPORT_WAIT_TIMEOUT = timedelta(minutes=config.report_wait_timeout).total_seconds
+        if(config and config.report_generation_max_retries and config.report_generation_max_retries > 5):
+            self.REPORT_GENERATION_MAX_RETRIES = config.report_generation_max_retries
```

Comment:
This won't be a CONSTANT anymore can you edit to become a variable and follow python convetion?

Reply:
Resolved